contrib/python-jsonschema-specifications/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "python-jsonschema-specifications"
+pkgver = "2023.12.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-hatch_vcs",
+    "python-hatchling",
+    "python-installer",
+]
+checkdepends = [
+    "python-referencing",
+    "python-pytest",
+]
+depends = ["python-referencing"]
+pkgdesc = "Json Schema meta-schemas and vocabularioes, exposed as a Registry"
+maintainer = "miko <mikoxyzzz@gmail.com>"
+license = "MIT"
+url = "https://github.com/python-jsonschema/jsonschema-specifications"
+source = f"{url}/releases/download/v{pkgver}/jsonschema_specifications-{pkgver}.tar.gz"
+sha256 = "48a76787b3e70f5ed53f1160d2b81f586e4ca6d1548c5de7085d1682674764cc"
+
+
+# get rid of this once pep517's check stuff is fixed up
+def pre_check(self):
+    whl = list(
+        map(
+            lambda p: str(p.relative_to(self.cwd)),
+            self.cwd.glob(self.make_install_target),
+        )
+    )
+
+    self.do(
```

Comment:
do we really need this venv nonsense for every single one of these? because it hasn't been needed for literally anything until now

Reply:
no, just the native-module ones. but you could make it the default for do_check and it would be an improvement like i explained yesterday

Reply:
> no, just the native-module ones.  

(or to be specific- it depends on the build-backend, setuptools keeps it in build/lib.* too. but maturin does not persist a whole pythonpath import on disk and only gives you a wheel. with a venv though you can always 'just install the wheel' and not care about any of the specifics, like how in some templates we add PYTHONPATH to the build/lib folder, etc)